examples/mandelbrot.cpp

```diff
@@ -228,7 +229,7 @@ int main()
     const float y1            = 1;
     const int maxIters        = 256;
 
-    alignas(64) std::array<int, width * height> buf;
+    alignas(64) std::vector<int> buf(width * height);
```

Comment:
`alignas(64)` does not affect heap allocation here, so you need to use

```
std::vector<int, xsimd::aligned_allocator<int, XSIMD_DEFAULT_ALIGNMENT>>;
```

instead

Reply:
Thank you